packages/twenty-front/src/modules/workflow/components/RightDrawerWorkflowEditStepContent.tsx

```diff
@@ -84,10 +85,25 @@ export const RightDrawerWorkflowEditStepContent = ({
     );
   }
 
-  return (
-    <WorkflowEditActionForm
-      action={stepDefinition.definition}
-      onActionUpdate={updateStep}
-    />
-  );
+  if (stepDefinition.type === 'action') {
+    if (stepDefinition.definition.type === 'CODE') {
```

Comment:
Let's use a switch, we will add more types soon

Reply:
can you use the type from `packages/twenty-front/src/modules/workflow/constants/Actions.ts`

Reply:
I will work on that tomorrow. Thanks.

Reply:
Fixed 👍